Library/Formula/node.rb

```diff
@@ -21,7 +22,7 @@ class Node < Formula
 
   depends_on :python => :build if MacOS.version <= :snow_leopard
   depends_on "pkg-config" => :build
-  depends_on "openssl" => :optional
```

Comment:
Although, again, I personally would advocate that this option shouldn't really be necessary, since the default configuration is perfectly fine. If it _is_ left as an option, `--with-brewed-openssl` would be clearer.